src/rdkafka_conf.c

```diff
@@ -829,7 +829,10 @@ static const struct rd_kafka_property rd_kafka_properties[] = {
 	  "Local message timeout. "
 	  "This value is only enforced locally and limits the time a "
 	  "produced message waits for successful delivery. "
-          "A time of 0 is infinite.",
+	  "A time of 0 is infinite. "
```

Comment:
mix of tabs and spaces. Use 8 whitespaces as tab indention.

Reply:
That's a thorough review, thanks !
I think and hope I have properly taken into account all the changes while preserving the individual commits (since they refer to past issues).
Only your last comment for the whitespace issue is mysterious to me. As it is now in the patch, the line "A time of 0 is infinite" seems to be indented exactly as all other lines in this file (i.e. vim tells me it starts at column 4-7), which I think was not the case before and had to be corrected.

Reply:
Right, this is one of those cases where the existing code uses the old tab-for-indent.
What I typically do is change the existing related statement to the new space-for-indent style.
But leave it as it is, it is fine :)